style.css

```diff
@@ -41,6 +52,38 @@ body {
   font-weight: bold;
   font-size: 32px;
   line-height: 44px;
+  margin: 3% auto;
+  margin-right: 10px;
+  margin-left: 10px;
+  margin-bottom: 20px;
+  width: 94%;
+  height: fit-content;
+  min-height: 94%;
+  border-radius: 12px;
+  padding: 16px;
+  zoom: 75%;
+}
+
+.see-pop {
+  display: flex;
+  justify-content: center;
+  align-content: center;
+  flex-direction: row;
+  gap: 5%;
+  padding-top: 20px;
+}
+
+.see-pop > .press {
+  min-width: 155px;
+  height: 48px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  align-content: center;
+  background-color: #fff;
+  border: 1px solid #6070ff;
+  border-radius: 12px;
+  padding-top: 10px;

```

Comment:
[OPTIONAL] Good job handling the issues pointed out by my colleague. However, i feel there are tiny improvements that can still be made to the tech tags and buttons. I feel their sizes are currently too small. For legibility and accessibility's sake, I recommend increasing their sizes.